plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -196,15 +200,11 @@ private FileCollection parseTargetsIsExclude(Object[] targets, boolean isExclude
 		} else if (targets.length == 1) {
 			return parseTargetIsExclude(targets[0], isExclude);
 		} else {
```

Comment:
I guess we don't need any of these `targets.length` special cases anymore. Your call whether we keep them or just loop over the array every time.

Reply:
Changed my mind, I think it's a bit easier to read to keep the special cases.